test/e2e/antreapolicy_test.go

```diff
@@ -2504,7 +2504,9 @@ func printResults() {
 			}
 			fmt.Printf("\tStep %s on port %d, duration %d seconds, result: %s\n",
 				step.Name, step.Port, int(step.Duration.Seconds()), result)
-			fmt.Printf("\n%s\n", comparison.PrettyPrint("\t\t"))
+			if testFailed {
```

Comment:
should this be `if testFailed` or `if wrong`?

Reply:
Thanks for catching. Updated.